src/rpdk/core/contract/resource_client.py

```diff
@@ -109,6 +110,9 @@ def __init__(
         self._overrides = overrides
         self._update_schema(schema)
         self._inputs = inputs
+        self._timeout_in_seconds = (
+            30 if timeout_in_seconds is None else int(timeout_in_seconds)
```

Comment:
maybe instead of `timeout_in_seconds=None` we should have `timeout_in_seconds=30`?
then code could be just `self._timeout_in_seconds = int(timeout_in_seconds)`

Reply:
I wanted to set the timeout_in_seconds parameter to None so that we can keep this at the very end.